pkg/plugins/config/config.go

```diff
@@ -26,6 +26,8 @@ type PluginManagementCfg struct {
 
 	AngularSupportEnabled  bool
 	HideAngularDeprecation []string
+
+	GcomToken string
```

Comment:
Can we call this `GrafanaComAPIToken`? And place it below `GrafanaComAPIURL`, that will make it clearer that both things are coupled.